### PR TITLE
Enable 'adapted args' warnings globally

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: akka-actor/src/main/scala/akka/io/dns/internal/DnsClient.scala [hunk 1]
```diff
@@ -82,21 +82,21 @@ import scala.concurrent.duration._
     case Question4(id, name) =>
       log.debug("Resolving [{}] (A)", name)
       val msg = message(name, id, RecordType.A)
-      inflightRequests += (id -> (sender(), msg))
+      inflightRequests += (id -> ((sender(), msg)))
```
Comment (patriknw) [inline akka-actor/src/main/scala/akka/io/dns/internal/DnsClient.scala]: is this needed if written as `(sender() -> msg)`?

**Review by patriknw — APPROVED** [review]
LGTM, with syntax question

File: akka-actor/src/main/scala/akka/io/dns/internal/DnsClient.scala (hunk 1 above)
Comment (hepin1989) [reply akka-actor/src/main/scala/akka/io/dns/internal/DnsClient.scala]: better

Comment (raboof) [reply akka-actor/src/main/scala/akka/io/dns/internal/DnsClient.scala]: That's indeed nicer, thx!

**Comment by patriknw** [comment]
thx

**Review by johanandren — APPROVED** [review]

**Pull request merged**